Adyen/HttpClient/HttpClientExtension.cs

```diff
@@ -0,0 +1,56 @@
+#region License
+// /*
```

Comment:
We do not want to use these ASCII headers anymore

Reply:
Good point, I started removing them in these PR's but @AlexandrosMor suggested I remove all of them in a separate PR.